source/documentation/query/javascript-functions.md

```diff
@@ -97,9 +106,7 @@ For example, "some words to process " becomes "someWordsToProcess".
     // Words to be combined are separated by a space in the string.
     
     function toCamelCase(str) {
-        return str.split(' ')
-    	.map(cc)
-    	.join('');
+        return str.split(' ').map(cc).join('');
```

Comment:
I'm fine either way. But if that's JS, the previous way is now more common, I think (although for shorter chained calls like this one I think it's almost 50% 50%) - https://eslint.org/docs/latest/rules/newline-per-chained-call

Reply:
(but it really doesn't matter... it's only for readability, same reason why it's becoming more common to drop the `;` too, and rely on ECMAScript implementations ability to perform the automatic semicolon insertion ([ASI](https://tc39.es/ecma262/#sec-automatic-semicolon-insertion)) - parsing JS must be tough)

Reply:
Chained-calls realigned.